clients/src/main/java/org/apache/kafka/common/security/scram/internals/ScramMechanism.java

```diff
@@ -40,11 +48,13 @@ public enum ScramMechanism {
         MECHANISMS_MAP = Collections.unmodifiableMap(map);
     }
 
-    ScramMechanism(String hashAlgorithm, String macAlgorithm, int minIterations) {
+    ScramMechanism(byte type, String hashAlgorithm, String macAlgorithm, int minIterations, int maxIterations) {
```

Comment:
can you use standard indentation here
```
Foo(
  bar
  baz
  quux
) {
}
```

Reply:
Fixed. Note that checkstyleMain sets java indentation here at 4 spaces not 2.